packages/doc/package.json

```diff
@@ -14,6 +14,11 @@
     "@mdx-js/loader": "^1.4.5",
     "@mdx-js/mdx": "^1.4.5",
     "@mdx-js/react": "^1.4.5",
+    "@types/classnames": "^2.3.1",
+    "@types/node": "^18.11.17",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.9",
+    "@types/react-helmet": "^6.1.6",
```

Comment:
Shouldn't these `@types` and `typescript` be on `devDependencies` instead? They will be used only in development and should not be part of the final bundle